### DISCUSSION TITLE
3dtiles+transform

### DISCUSSION THREAD

**Discussion by calloffate** [visgl/deck.gl#6634] [Q&A]
How do I move, rotate and scale the loaded 3dtiles?
I want to do it via modelMatrix in the callback function, but the effect is not correct
![image](https://user-images.githubusercontent.com/43996189/153127501-288bc247-cbf2-4cf9-ad6f-8c86f44c9b11.png)

**Comment by ibgreen** [comment]
Would have to check exactly how the tileset's modelMatrix is used, but note that the layer itself takes a `modelMatrix` prop that should be in the coordinate system you want, maybe try something like:

```
new Tile3DLayer ({
   ...
   modelMatrix: new Matrix4().translate([0, 0, 100]),
  ....

**Reply by calloffate** [reply]
Thank you for your reply.

**Reply by eoudejans** [reply]
@calloffate does the proposed solution work for you? 

Proposed solution does not seem to work in my use case:
```
new Tile3DLayer ({
   ...
   modelMatrix: new Matrix4().translate([0, 0, 100]),
  ....
```

See for a working tileset in the Netherlands, Rotterdam: https://www.3drotterdam.nl/datasource-data/d0c755ef-ba1b-43fc-bc44-24fc7bb152ce/tileset.json

**Reply by grahambates-code** [reply]
I don't this is possible, currently. I think the model matrix needs to be passed to each sub layer in Tile3DLayer. If anyone can provide any hints on the viability of this being the answer I could try making a PR
